tools/mysql/schema-update-8.sql

```diff
@@ -5,7 +5,7 @@
 # This script upgrade DB schema from version 7 to version 8
 
 ALTER TABLE environs ADD COLUMN stage_type VARCHAR(32) NOT NULL DEFAULT 'PRODUCTION';
-ALTER TABLE groups MODIFY group_name VARCHAR(128) NOT NULL;
+ALTER TABLE groups_and_roles MODIFY group_name VARCHAR(128) NOT NULL;
```

Comment:
groups_and_roles contains ldap group to role mapping and so does not require this ALTER query.

you are correct about non-exisitng groups table though.

Reply:
thanks @ankilosaurus. Should I modify this PR to just take out the `groups` alter?